character-controller.js

```diff
@@ -1168,94 +1088,275 @@ class LocalPlayer extends UninterpolatedPlayer {
     camera.position.sub(localVector.copy(cameraOffset).applyQuaternion(camera.quaternion));
     camera.updateMatrixWorld();
   } */
-  
-  pushPlayerUpdates() {
-    this.playersArray.doc.transact(() => {
-      /* if (isNaN(this.position.x) || isNaN(this.position.y) || isNaN(this.position.z)) {
-        debugger;
-      } */
-      this.playerMap.set('position', this.position.toArray(localArray3));
-      this.playerMap.set('quaternion', this.quaternion.toArray(localArray4));
-    }, 'push');
-
-    this.appManager.updatePhysics();
+  packed = new Float32Array(8);

```

Comment:
We are initializing in `this` already, so initializing more here is inconsistent. Can we put all properties in the same place?